index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300&family=Poppins:wght@300&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+  <div class="portfolio">
+    <header>
+      <nav class="navbar">
+        <div class="navbottom">
+          <div class="logo">
+            <h2>Carlyle Codes</h2>
+          </div>
+          <div class="menu">
+            <i class="fas fa-bars"></i>
+          </div>
+        </div>
+      </nav>
+      <section class="headline">
+        <div class="content">
+          <h1>Hey There. I'm Carlyle</h1>
+          <p class="h3">I am a Software Developer</p>
+          <p class="h5">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <ul class="social-links">
+          <li><a href="#"><img class="box one" src="images/Vector@2x.png" alt=""></a></li>
+          <li><a href="#"><i class="fab fa-linkedin-in two box"></i></a></li>
+          <li><a href="#"><img class="box three" src="images/Vector.png" alt="two fingers, peace sign"></a></li>
+          <li><a href="#"><img class="box four" src="images/twitter.png" alt="twitter icon"></a></li>
+          <li><a href="#"><img class="box five" src="images/m-icon.png" alt="M"></a></li>
+        </ul>
+      </section>
+      <div class="headerbg">
+        <img src="images/Illustration.png" alt="illustration image">
+      </div>
```

Comment:
- [ ] Decoration images should be inserted using the CSS property `background-image`, and not as inline elements. Please change this section to include the image as a background.